poke-cdn.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Force a cdn broker instance into the `provisioned` state so that it can be deleted
+# Usage: `DOMAIN="some.domain.gov" ./poke-cdn.sh`
+
+set -e
+set -u
+set -x
+
+if ! cf plugins | grep "connect-to-service"; then
+  echo "connect-to-service plugin not found; install at https://github.com/18F/cf-service-connect#local-installation"
+  exit 1
+fi
+
+creds="creds.out"
+
+cf target -o ${ORG:-cloud-gov} -s ${SPACE:-services}
+cf connect-to-service --no-client ${APP:-cdn-broker} ${DATABASE:-rds-cdn-broker} > ${creds} &
```

Comment:
Can the script from this point on be simplified to just this oneliner?

`echo "update routes set state = 'provisioned' where domain_external = '${DOMAIN}' and state = 'provisioning';" | cf connect-to-service ${APP:-cdn-broker} ${DATABASE:-rds-cdn-broker} > ${creds}`